### DISCUSSION TITLE
Can you deploy a branch to a subdomain with a different static folder?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [vercel/community#67] [Help] [answered]
Hi, my project consists of a Next.js app and a Storybook static folder which I'd like to publish.  Is it possible to do the following?

- Deploy the Next.js app on the production and preview branches, while also
- Deploying only a `storybook` branch to a specific subdomain but from a separate static folder, eg `/storybook-static` 

I understand you can attach a branch to a subdomain. However I don't see an option to specify a separate output directory.

Thanks

**Comment by Pieparker** [comment, marked as answer]
Finding it hard to picture your use case and setup, if I'm honest 😛! However, from my read, it may be simpler to launch two separate projects. You may also get away with updating your build script and writing something suitable for your use case using `bash`.

**Reply by dungle-scrubs** [reply]
Thanks, I'm realizing that my question wasn't entirely clear.

What I ended up doing was what you suggested.  I have one product folder (and therefore one repo), and in that project I have a Next.js app and a Storybook app, so there are two output directories. 

After some tinkering with Next.js, I realized that you can't have a separate build configuration in the same project for a different branch (I wanted to have a single `storybook` branch that deployed to the `/storybook-static` directory while all other branches deployed the Next.js deploy directory).

Now I have two projects set up in Vercel linked to the same repo.  One project deploys Next.js, while the other deploys Storybook.

The Next.js app exists on the main domain, while the Storyboook app exists on its own subdomain (storybook.domain.com)

The only downside I've experienced so far is when using Vercel CLI, I need to unlink and re-link the same repo to different projects when I need to use it.

**Reply by Pieparker** [reply]
Also sounds like https://turborepo.org ([repo](https://github.com/vercel/turborepo)) could be a big lift for you, by the way!
